pkg/platform/packet/packet.go

```diff
@@ -236,6 +237,22 @@ func createTerraformConfigFile(cfg *config, terraformPath string) error {
 		// TODO: Render manually instead of marshaling.
 		return fmt.Errorf("marshaling management CIDRs: %w", err)
 	}
+
+	if len(cfg.NodePrivateCIDR) != 0 {
+		cfg.NodePrivateCIDRs = append(cfg.NodePrivateCIDRs, cfg.NodePrivateCIDR)
+	}
+
+	cfg.NodePrivateCIDRs = findUnique(cfg.NodePrivateCIDRs)
```

Comment:
I'd throw an error if there are duplicates, it seem like misconfiguration to me.

Reply:
I think it is harmless, since we are taking care of removing the duplicates. A win at UX front.

Reply:
Ignoring possible misconfiguration does not sound like good UX to me, as someone may for example mis-type the variable name and miss including other CIDR they meant.